yaksh/templates/base.html

```diff
@@ -30,40 +32,45 @@
 
   {% block css %}
   {% endblock %}
-  <script language="JavaScript" type="text/javascript" src="{{ URL_ROOT }}/static/yaksh/js/jquery-1.9.1.min.js"></script>
-  <script src="{{ URL_ROOT }}/static/yaksh/js/bootstrap.min.js"></script>
+
+            <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script> 
```

Comment:
I appreciate that there is an sub resource integrity check with the CDN. However, I would suggest having the JS script as a part of the repo, instead of calling it through the CDN.

Reply:
Done